.github/workflows/macos_meson.yml

```diff
@@ -165,8 +165,7 @@ jobs:
         sudo xcode-select -s /Applications/Xcode_15.2.app
         
         git submodule update --init
-        # for some reason gfortran is not on the path
-        GFORTRAN_LOC=$(brew --prefix gfortran)/bin/gfortran 
+        GFORTRAN_LOC=$(which gfortran-13)
```

Comment:
I think any issues you may have experienced in the past came from calling `which gfortran` instead of `which gfortran-{version}`. There are three versions guaranteed to exist under these aliases by GitHub (see [documentation](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md#installed-software)).